lib/msal-angular/package.json

```diff
@@ -1,119 +1,57 @@
-    "karma-spec-reporter": "0.0.32",
-    "karma-verbose-reporter": "0.0.6",
-    "karma-webpack": "^3.0.0",
-    "msal": "^1.4.2",
-    "ng-packagr": "^9.0.2",
-    "phantomjs-polyfill": "0.0.2",
-    "reflect-metadata": "^0.1.3",
-    "rxjs": "^6.5.4",
-    "shx": "^0.2.2",
-    "source-map-loader": "^0.2.3",
-    "ts-loader": "^2.0.3",
-    "tslib": "^1.11.1",
-    "typedoc": "^0.14.2",
-    "typescript": "^3.6.5",
-    "typings": "^2.1.1",
-    "webpack": "3.6.0",
-    "zone.js": "^0.8.10"
-  },
-  "directories": {
-    "test": "tests"
-  },
+  "version": "2.0.0-alpha.0",
   "scripts": {
-    "clean": "shx rm -rf dist ref",
-    "doc": "npm run doc:generate && npm run doc:deploy",
-    "doc:generate": "typedoc --mode modules --excludePrivate --excludeProtected --out ./ref ./src/ --gitRevision dev",
-    "doc:deploy": "gh-pages -d ref -a -e ref/msal-angular",
-    "build:modules": "ng-packagr -p package.json",
-    "build": "npm run clean && npm run build:modules",
-    "deploy": "npm run build && npm publish dist && npm run doc",
-    "test": "karma start karma.conf.js",
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
+    "test": "ng test",
     "lint": "cd ../../ && npm run lint:angular",
     "lint:fix": "npm run lint -- -- --fix",
-    "test:verify-build": "npm run build:modules && npm test",
-    "endToEnd": "cd tests/endToEndTests && npm install && node testRunner.js",
-    "prepack": "npm run build"
+    "e2e": "ng e2e"
   },
-  "description": "Microsoft Authentication Library for Angular",
+  "main": "./dist/bundles/azure-msal-angular.umd.js",
+  "typings": "./dist/azure-msal-angular.d.ts",
+  "private": true,
   "dependencies": {
     "minimatch": "^3.0.4",
     "path": "^0.12.7"
+  },
+  "devDependencies": {
+    "@angular-devkit/build-angular": "~0.1000.8",
+    "@angular-devkit/build-ng-packagr": "~0.1000.8",
+    "@angular/animations": "~10.0.9",
+    "@angular/cli": "~10.0.6",
+    "@angular/common": "~10.0.9",
+    "@angular/compiler": "~10.0.9",
+    "@angular/compiler-cli": "~10.0.9",
+    "@angular/core": "~10.0.9",
+    "@angular/forms": "~10.0.9",
+    "@angular/platform-browser": "~10.0.9",
+    "@angular/platform-browser-dynamic": "~10.0.9",
+    "@angular/router": "~10.0.9",
+    "@azure/msal-browser": "^2.5.0",
+    "@types/jasmine": "~3.5.0",
+    "@types/jasminewd2": "~2.0.3",
+    "@types/node": "^12.11.1",
+    "jasmine-core": "~3.5.0",
+    "jasmine-spec-reporter": "~5.0.0",
+    "karma": "~5.0.0",
+    "karma-chrome-launcher": "~3.1.0",
+    "karma-coverage-istanbul-reporter": "~3.0.2",
+    "karma-jasmine": "~3.3.0",
+    "karma-jasmine-html-reporter": "^1.5.0",
+    "ng-packagr": "^10.0.0",
+    "protractor": "~7.0.0",
+    "rxjs": "~6.5.5",
+    "ts-node": "~8.3.0",
+    "tslib": "^2.0.0",
+    "typescript": "~3.9.5",
+    "zone.js": "~0.10.3"
+  },
+  "peerDependencies": {
+    "@angular/common": "^10.0.14",
+    "@angular/core": "^10.0.14",
+    "@azure/msal-browser": "^2.4.0"
```

Comment:
I think `rxjs": "^6.0.0` should be added as a peer dep.

Reply:
Do we still keep it as a dev dependency?

Reply:
Yeah, needs to be both.

Reply:
Great. Changed.